### PR TITLE
iscsi: limit to cleanup all

### PR DESCRIPTION
**Pull request by hsmj1412**
Suppose there are 2 iscsi, but the case just want to cleanup one
of them. To use confirm=True will break the other iscsi config.

Signed-off-by: Junxiang Li <junli@redhat.com>

### CODE REVIEW

**Comment by kylazhang** [comment]
Can we add some flag to enable/disable the targetcli clear?

**Comment by kylazhang** [comment]
@prudmirya what's your suggestion

**Comment by chunfuwen** [comment]
@kylazhang ,this fail more than 100 automation cases, we may merge them firstly.

**Comment by balamuruhans** [comment]
> Can we add some flag to enable/disable the targetcli clear?

@chunfuwen I would prefer @kylazhang's suggestion instead of removing the targetcli clear part.
by default let it be disabled.

**Comment by chunfuwen** [comment]
@hsmj1412 , please add default parameter to cleanup(self) in iscsi.py with below:
cleanup(self, comfirmed = False).
and comfirmed == True,then do targetcli clear part operation

**Comment by hsmj1412** [comment]
@chunfuwen Done

File: virttest/iscsi.py [hunk 1]
```diff
@@ -335,7 +335,7 @@ def logout(self):
         if self.logged_in():
             iscsi_logout(self.target)
 
-    def cleanup(self):
+    def cleanup(self, comfirmed=False):
         """
         Clean up env after iscsi used.
```
Comment (chunfuwen) [inline virttest/iscsi.py]: comfirmed --> confirmed
Moreover, add doc string for this parameter

Comment (hsmj1412) [reply virttest/iscsi.py]: Done

**Comment by kylazhang** [comment]
LGTM

**Comment by kylazhang** [comment]
@chunfuwen @balamuruhans please help to confirm, thanks

**Comment by chunfuwen** [comment]
@balamuruhans , please help merge it since it is urgent

**Comment by balamuruhans** [comment]
Let us merge this.

**Pull request merged**